download_test_data.sh

```diff
@@ -52,6 +58,28 @@ if ! wget \
     echo "Problem in downloading the test data set (calibrated with missing trees) for real data."
 fi
 
+if ! wget \
+    -i test_data_real_missing_prescaler_trigger.txt \
+    --user="$TEST_DATA_USER" \
+    --password="$TEST_DATA_PASSWORD" \
+    --no-check-certificate \
+    --no-verbose \
+    --timestamping \
+    --directory-prefix=test_data/real/calibrated/missing_prescaler_trigger; then
+    echo "Problem in downloading the test data set (calibrated with missing prescaler and trigger trees) for real data."
+fi
+
+if ! wget \
+    -i test_data_real_missing_arrays.txt \
+    --user="$TEST_DATA_USER" \
+    --password="$TEST_DATA_PASSWORD" \
+    --no-check-certificate \
+    --no-verbose \
+    --timestamping \
+    --directory-prefix=test_data/real/calibrated/missing_arrays; then
+    echo "Problem in downloading the test data set (calibrated with missing prescaler and trigger arrays) for real data."
+fi
+
```

Comment:
taking into account that we already have 5 of those wget statements we could instead make a list of .txt files and associated directory prefix  and loop over them

Reply:
Done.